unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/ProfanityFiltering/ProfanityFilterSharedInstances.cs

```diff
@@ -1,13 +1,16 @@
 ﻿public static class ProfanityFilterSharedInstances
```

Comment:
It's better to use a contained composite class instead of static to avoid state corruption between tests and lifecycle anomalies.

Reply:
I agree, but I would change that on another PR